entrypoint.sh

```diff
@@ -1,7 +1,7 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
An alternative here would be to install Bash. But if this is all we need to do so support standard Bourne shell...